lib/block_view_creator_coin_test.go

```diff
@@ -211,7 +211,7 @@ func _helpTestCreatorCoinBuySell(
 		// If no UtxoView is passed, use a new one to run our checks.
 		if utxoView == nil {
 			var err error
-			utxoView, err = NewUtxoView(db, params, nil)
+			utxoView, err = NewUtxoView(db, params, nil, chain.snapshot)
```

Comment:
Yeah, I'm considering that too. Though honestly, we don't call NewUtxoView that much in main code, we only do that a lot in tests. It does make sense to wrap these structs when we pass it to db_utils functions because there we pass snapshot almost everywhere and it seems suboptimal. I'll keep thinking about this.